get_postgis_func.py

```diff
@@ -0,0 +1,548 @@
+import json
```

Comment:
This should be an `edb` tool, so move it into `edb/tools` and make it a proper command